src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaCandidateSearchQuery.php

```diff
@@ -95,6 +100,19 @@ public function setEmail($email)
     }
 
     /**
+     * @param array $secondFactorTypes
+     *
+     * @return void
+     */
+    public function setSecondFactorTypes(array $secondFactorTypes)
+    {
+        Assert\Assertion::allString($secondFactorTypes);
+        Assert\Assertion::allNotEmpty($secondFactorTypes);
```

Comment:
Could you provide some feedback if the assertion's requirement is not met? For an idea, look at how the other properties are set.